dallinger/command_line.py

```diff
@@ -529,11 +509,9 @@ def deploy(verbose, app):
     config.load_config()
 
     # Set the mode.
-    config.set("Experiment Configuration", "mode", "deploy")
-    config.set("Server Parameters", "logfile", "-")
-
-    # Ensure that we do not launch in sandbox mode.
-    config.set("Shell Parameters", "launch_in_sandbox_mode", "false")
+    config.extend({"mode": u"sandbox",
+                   "logfile": u"-",
+                   "launch_in_sandbox_mode": False})
```

Comment:
These config changes are all written out in the call to `deploy_sandbox_shared_setup` here: https://github.com/Dallinger/Dallinger/blob/aed0d63663c465df1e60975884b497adfbdee91e/dallinger/command_line.py#L443